css-layouts/flexbox/src/exercises/exercise2/index.js

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
Nit: Make sure that the lines of text here appear appropriately on laptop screens in CodeSandbox